jenkins_pipelines/environments/manager-TEST-reportdb-hub-tests

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env groovy
```

Comment:
I would use if you don't mind, `manager-TEST-Orion-acceptance-tests-reportdb-hub`